trigger/testTimer/README.md

```diff
@@ -0,0 +1,80 @@
+<!--
+title: CSV Timer
+
+-->
+# Timer Trigger
+This trigger provides your flogo application the ability to get data from a CSV file after a particular interval for an action 
+
+## Installation
+
+```bash
+flogo install github.com/skothari-tibco/csvtrigger
+```
+
+## Metadata
+```json
+{
+  "handler": {
+    "settings": [
+      {
+        "name": "filePath",
+        "type": "string"
+      },
+      {
+        "name": "repeatInterval",
+        "type": "string"
+      }
+    ]
+  }
+}
+```
+### Details
+
+#### Settings:
+| Setting  | Required | Description |
+|:---------|:---------|:------------|
+| control  | false    | Enable control of trigger
+| port     | false    | Port number on which control api is called (/control/resume, /control/pause, /control/restart)
```

Comment:
Add a section in the README.md that describes the control aspect.  I wouldn't list those here.  In that section also explain specifying handlerId in order to control a specific handler.